deploy/cheesefork.css

```diff
@@ -25,6 +25,10 @@
     visibility: hidden !important;
 }
 
+#top-navbar-share-link {
+    display: none;
```

Comment:
It's unnecessary for the other menu items. For consistency, please implement the new one in the same way.